include/auxkernels/ModeratorHeatSourceTransientAux.h

```diff
@@ -9,6 +9,15 @@ class ModeratorHeatSourceTransientAux;
 template <>
 InputParameters validParams<ModeratorHeatSourceTransientAux>();
 
+/**
+ * When a reactor runs, gamma rays are emitted in extraordinary quantity.
+ * These gammas tend to heat the graphite in thermal MSRs, as experienced at the MSRE,
+ * and make the graphite's steady temperature quite a bit hotter than then the salt.
+ * ORNL found that gamma heating in graphite is nearly uniform over the core since gammas have
+ * a long mean free path. In addition, the gamma heating is proportional to the average fission
+ * heat. This kernel computes local gamma heating as a function of average fission heat and a
+ * user-defined proportionality factor (usually between 2 and 10 percent).
```

Comment:
I changed gamma to be a function which can allow definition of gamma heating that is not uniform (i.e. maybe you choose gamma to be sinusoidal in both radial and axial directions). I think this doxygen string is fine, however, unless you want to add that detail.